src/api/Opensmt.h

```diff
@@ -30,8 +31,7 @@ typedef enum
   , qf_auflira    // Arrays + UF + (LIA or LRA)
 } opensmt_logic;
 
-class Opensmt
-{
+class Opensmt {
```

Comment:
It feels more like a solver to me, but it would be another "solver" in the namespace next to `MainSolver` and others. I consider the name of the class appropriate since it somehow represents the whole "product". Maybe be can include it in the namespace - it seems redundant, but it would be at least consinstent with the current codebase (e.g. `ite::IteHandler`, `pterms/Pterm.h` etc.)

Reply:
I would feel better if it is in the namespace.